.ci/install-clang-devel.sh

```diff
@@ -32,26 +32,26 @@ add-apt-repository -y "deb-src http://apt.llvm.org/unstable/ llvm-toolchain-${CL
 apt-get update -y
 
 apt-get install -y --no-install-recommends \
-    clang-${CLANG_VERSION} \
-    clangd-${CLANG_VERSION} \
-    clang-format-${CLANG_VERSION} \
-    clang-tidy-${CLANG_VERSION} \
-    clang-tools-${CLANG_VERSION} \
-    lldb-${CLANG_VERSION} \
-    lld-${CLANG_VERSION} \
-    llvm-${CLANG_VERSION}-dev \
-    llvm-${CLANG_VERSION}-tools \
-    libomp-${CLANG_VERSION}-dev \
-    libc++-${CLANG_VERSION}-dev \
-    libc++abi-${CLANG_VERSION}-dev \
-    libclang-common-${CLANG_VERSION}-dev \
-    libclang-${CLANG_VERSION}-dev \
-    libclang-cpp${CLANG_VERSION}-dev \
-    libunwind-${CLANG_VERSION}-dev
+    "clang-${CLANG_VERSION}" \
+    "clangd-${CLANG_VERSION}" \
+    "clang-format-${CLANG_VERSION}" \
+    "clang-tidy-${CLANG_VERSION}" \
+    "clang-tools-${CLANG_VERSION}" \
+    "lldb-${CLANG_VERSION}" \
+    "lld-${CLANG_VERSION}" \
+    "llvm-${CLANG_VERSION}"-dev \
+    "llvm-${CLANG_VERSION}"-tools \
+    "libomp-${CLANG_VERSION}"-dev \
+    "libc++-${CLANG_VERSION}"-dev \
+    "libc++abi-${CLANG_VERSION}"-dev \
+    "libclang-common-${CLANG_VERSION}"-dev \
+    "libclang-${CLANG_VERSION}"-dev \
+    "libclang-cpp${CLANG_VERSION}"-dev \
+    "libunwind-${CLANG_VERSION}"-dev
```

Comment:
```suggestion
    "llvm-${CLANG_VERSION}-dev" \
    "llvm-${CLANG_VERSION}-tools" \
    "libomp-${CLANG_VERSION}-dev" \
    "libc++-${CLANG_VERSION}-dev" \
    "libc++abi-${CLANG_VERSION}-dev" \
    "libclang-common-${CLANG_VERSION}-dev" \
    "libclang-${CLANG_VERSION}-dev" \
    "libclang-cpp${CLANG_VERSION}-dev" \
    "libunwind-${CLANG_VERSION}-dev"
```

Please don't leave quotes in the middle of these words, as I asked in https://github.com/microsoft/LightGBM/pull/6504#discussion_r1651475750.